### DISCUSSION TITLE
Is Tailscale LXC script down?

### DISCUSSION THREAD

**Discussion by pager8** [tteck/Proxmox#3370] [General]
I'm trying to run Tailscale LXC script but unfortunately it crashes at the beginning. The problem is similar like in this [discussion](https://github.com/tteck/Proxmox/discussions/2267) but my DNS is fine (1.1.1.1. and 8.8.8.8). I can't find details of the scripts on GitHub, no mentions in changelog as well. Is it discontinued?

**Comment by Mellowlynx** [comment]
The Script is fine, try to ping GitHub from the node you want to install the LXC on.
Also, what is the error you get??

**Reply by pager8** [reply]
GitHub pinging is ok:

`27 packets transmitted, 27 received, 0% packet loss, time 26040ms`

There is no specific error shown in the shell. Script is asking to choice CT, printing "Installing Tailscale..."  and after few seconds returns to default state (so I can type commands).

`Installing Tailscale...
root@proxmox:~# `

**Comment by pager8** [comment]
There was a problem with default Debian/Ubuntu LXC in my Proxmox. Creating Debian LXC via tteck's script and then running Tailscale script solved the problem.
